baselines/dev/create_baselie.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Let's rename the file using a `-` instead of an `_`, and also `baseline` instead of `baselie` 🤣